src/api/app/models/project.rb

```diff
@@ -238,10 +238,11 @@ def subprojects
 
   def siblingprojects
     parent_name = parent.try(:name)
-    return [] unless parent_name
-    Project.where('name like (?) and name != (?)', "#{parent_name}:%", name).order(:name).select do |sib|
+    return Project.none unless parent_name
+    projects_id = Project.where('name like (?) and name != (?)', "#{parent_name}:%", name).order(:name).select do |sib|
       sib if parent_name == sib.possible_ancestor_names.first
-    end
+    end.pluck(:id)
+    Project.where(id: projects_id)
```

Comment:
I think we should add a `TODO` here. I think it is fine to merge the server pagination with this ugly fix, but we should refactor this.

Reply:
I agree, but isn't it evident that this is ugly and should be changed? I don't think a fixme wouldn't help here much.